Assets/Huawei/Demos/Game/GameAddictionPreventionDemo/GameAddictionPreventionDemoManager.cs

```diff
@@ -0,0 +1,127 @@
+using HmsPlugin;
+using HuaweiMobileServices.Base;
+using HuaweiMobileServices.Game;
+using HuaweiMobileServices.Id;
+using HuaweiMobileServices.Utils;
+using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class GameAddictionPreventionDemoManager : MonoBehaviour
+{
+    private const string TAG = "[HMS] GameAddictionPreventionDemoManager";
+
+    [SerializeField]
+    Text display, btn;
```

Comment:
`btn `name can be more readable and should have a `_ `sign at the beginning according to the naming stands for display and button change.

Like maybe `_display `and `_buttonText `